src/common/options/osd.yaml.in

```diff
@@ -133,6 +133,51 @@ options:
   - osd_recovery_sleep
   flags:
   - runtime
+- name: osd_recovery_sleep_degraded
+  type: float
+  level: advanced
+  desc: Time in seconds to sleep before next recovery or backfill op when pg is degraded.
+    This setting overrides _ssd, _hdd, and _hybrid if non-zero.
+  fmt_desc: Time in seconds to sleep before the next recovery or backfill op when pg
+    is degraded. Increasing this value will slow down recovery operation while client
+    operations will be less impacted.
+  default: 0
+  flags:
+  - runtime
+- name: osd_recovery_sleep_degraded_hdd
+  type: float
+  level: advanced
+  desc: Time in seconds to sleep before next recovery or backfill op for HDDs
+    when pg is degraded.
+  fmt_desc: Time in seconds to sleep before next recovery or backfill op
+    for HDDs when pg is degraded.
+  default: 0.1
+  flags:
+  - runtime
+- name: osd_recovery_sleep_degraded_ssd
+  type: float
+  level: advanced
+  desc: Time in seconds to sleep before next recovery or backfill op for SSDs
```

Comment:
Ditto

Reply:
I do wonder if at some point we should fully embrace an additional distinction between fast and slow SSDs, beyond the scope of this PR of course.  One might argue that mclock in theory should suffice for that distinction as we would need more than just the kernel’s rotational flag.